packages/api-specification/api/ssf.ts

```diff
@@ -0,0 +1,46 @@
+import {SystemAPI} from "./system";
+import {WindowsAPI} from "./windows";
+import {MessagesAPI} from "./messages";
+import {NotificationsAPI} from "./notifications";
+import {ApplicationAPI} from "./application";
+import {SSFInfo} from "./info";
+
+/**
+ * Describes the ssf object as exposed to clients applications
+ *
+ * SSF object groups different APIs, where specific API methods can be accessed using the corresponding sub-object, e.g.
+ * to access window management API :
+ *
+ * ssf.windows.open({name:'search', url: 'http://google.com'});
+ */
+export interface SSF {
+    /**
+     * Window management API - create, discover and manage windows
+     */
+    windows: WindowsAPI;
+
+    /**
+     * Messaging API - communicate with other windows
+     */
+    messages: MessagesAPI;
+
+    /**
+     * Notifications API - publish notifications to the user
+     */
+    notifications: NotificationsAPI;

```

Comment:
I guess the interesting thing here is that both within the browser and Electron container users will see two notification APIs:

```
new Notification(...)
new ssf.Notification(...)
```

This is probably unavoidable, but something that users need to be warned about.